varya/default-style/variables.css

```diff
@@ -46,28 +46,32 @@
 
 	/* Blocks */
     --button--border-radius: 4px;
-    --button--border-width: 2px;
+    --button--border-width: 1px;
     --button--color-text: var(--global--color-white);
-    --button--color-text-hover: var(--button--color-foreground);
+    --button--color-text-hover: var(--button--color-text);
     --button--color-background: var(--global--color-secondary);
     --button--color-background-hover: var(--global--color-secondary-hover);
+	  --button--color-background-active: var(--global--color-primary);
     --button--font-weight: normal;
     --button--padding-horizontal: 20px;
     --button--padding-vertical: 22px;
 
     --separator--width: 52px;
     --separator--height: 2px;
-    --separator--border-color: var(--global--color-border);
+    --separator--border-color: #757575;
```

Comment:
Drats, I got this one wrong. It should be `var(--global--color-border);` and then the the global variable should be `--global--color-border: #757575`